### ISSUE TITLE
Ability to scroll above first line (over the top)

### ISSUE THREAD

**Issue by Servail** [microsoft/vscode#85268]
tl;dr: read the title

Currently there's option to scroll beyond last line - it's good because it allows to keep the bottom code in center on screen and makes it comfortable to work with (which is perfectly cozy with centered layout). Then why there's no option to scroll above first line? When scroll is on top the code is obviously not in the center, and feels squishy. All it needs is some fake empty lines at the top of file (like with Find widget). Of course, there can be license to push the code down or I can paste a bunch of empty lines myself but hey, I don't want to put license in my every private file (its distracting too) and real empty lines just... seems wrong (and annoyng to paste every time).  

P.S.: Can't believe there's no option or even suggestion/discussion so far... so I think it's also a good chance for VSCode to shine up with a Brand New Feature®! ;) Put like if agree lol

**Comment by bradkrane** [comment]
I just got fed up enough with not being able to do this myself to look into it. I second this motion.

**Comment by bradkrane** [comment]
This one is similar to [#56174 ](https://github.com/Microsoft/vscode/issues/56174) and JavierReyes945 comment
